src/Microsoft.AspNetCore.Razor.Evolution/DefaultRazorRuntimeCSharpLoweringPhase.cs

```diff
@@ -332,6 +332,16 @@ internal override void VisitCreateTagHelper(CreateTagHelperIRNode node)
                     tagHelperVariableName);
             }
 
+            internal override void VisitAddPreallocatedTagHelperHtmlAttribute(AddPreallocatedTagHelperHtmlAttributeIRNode node)
```

Comment:
FYI will will need to remove all of these kinds of specialized nodes before we ship. This is ok as a temporary thing while we figure out the design in this space.